moveit_ros/moveit_servo/config/servo_parameters.yaml

```diff
@@ -125,6 +126,15 @@ servo:
       description: "Max joint angular/linear velocity. Only used for joint commands on joint_command_in_topic."
     }
 
+
+  angular_velocity_about_ee_frame: {
```

Comment:
This param name doesn't read like it's a boolean, but rather as the angular velocity itself.

Maybe `apply_twist_commands_about_ee_frame` or something?

Reply:
👍